project/CelebornBuild.scala

```diff
@@ -958,11 +962,10 @@ object MRClientProjects {
 
         (assembly / assemblyMergeStrategy) := {
           case m if m.toLowerCase(Locale.ROOT).endsWith("manifest.mf") => MergeStrategy.discard
+          // For netty-3.x.y.Final.jar
           case m if m.startsWith("META-INF/license/") => MergeStrategy.discard
-          case m if m == "META-INF/LICENSE.txt" => MergeStrategy.discard
-          case m if m == "META-INF/NOTICE.txt" => MergeStrategy.discard
-          case m if m == "LICENSE.txt" => MergeStrategy.discard
-          case m if m == "NOTICE.txt" => MergeStrategy.discard
+          case PathList(ps@_*) if ps.last == "LICENSE" || ps.last == "NOTICE" => MergeStrategy.preferProject
```

Comment:
```suggestion
          case "LICENSE" | "NOTICE" => MergeStrategy.discard
          case PathList(ps@_*) if ps.last == "LICENSE" || ps.last == "NOTICE" => MergeStrategy.preferProject          
```

we should discard the NOTICE/LICENSE files from the `scala-library` for mr-shade client.

Reply:
```
$ ./build/sbt -Pmr celeborn-client-mr-shaded/assembly
$ jar tf client-mr/mr-shaded/target/scala-2.12/celeborn-client-mr-shaded_2.12-0.4.0-SNAPSHOT.jar | grep -i 'license\|notice'
LICENSE
META-INF/LICENSE
META-INF/NOTICE
META-INF/licenses/
META-INF/licenses/LICENSE-protobuf.txt
NOTICE
```

Reply:
Thanks for finding this problem.
I have now changed to use `preferProject` only for `META-INF/LICENSE` and `META-INF/NOTICE`.